src/napari_graph/interops.py

```diff
@@ -0,0 +1,109 @@
+from typing import Any
+
+import networkx as nx
+import numpy as np
+import pandas as pd
+
+from napari_graph.base_graph import BaseGraph
+from napari_graph.directed_graph import DirectedGraph
+from napari_graph.undirected_graph import UndirectedGraph
+
+
+def from_networkx(graph: nx.Graph) -> BaseGraph:
+    """Convert a NetworkX graph into a napari-graph UndirectedGraph or DirectedGraph.
+
+    Parameters
+    ----------
+    graph : nx.Graph
+        The NetworkX graph to be converted.
+
+    Returns
+    -------
+    BaseGraph
+        An equivalent napari-graph UndirectedGraph or DirectedGraph.
+    """
+
+    coords_dict = nx.get_node_attributes(graph, "pos")
+    if len(coords_dict) > 0:
+        coords_df = pd.DataFrame.from_dict(coords_dict, orient="index")
+    else:
+        coords_df = None
+
+    edges = graph.edges
+    if len(edges) > 0:
+        edges = np.atleast_2d(edges)
+
+    if graph.is_directed():
+        out_graph = DirectedGraph(edges, coords_df)
+    else:
+        out_graph = UndirectedGraph(edges, coords_df)
+
+    return out_graph
+
+
+def to_networkx(graph: BaseGraph) -> nx.Graph:
```

Comment:
Here is what I meant by pass through: this should be:

```suggestion
def to_networkx(graph: BaseGraph | nx.Graph) -> nx.Graph:
```

and below you would say `if isinstance(graph, nx.Graph): return graph`.

Reply:
I made this a static method, so I won't make it pass through.
The function `to_napari_graph` that accepts "any" graph is pass-through. How does that sound?